packages/pilot/src/containers/TransactionsList/tableColumns.js

```diff
@@ -18,8 +19,14 @@ const convertPaymentValue = property => pipe(
   formatCurrency
 )
 
+const getCustomerName = ifElse(
+  path(['customer', 'name']),
+  path(['customer', 'name']),
+  path(['card', 'holder_name'])
+)
+
```

Comment:
O que acha de usarmos o either aqui?
```suggestion
const getCustomerName = either(
  path(['customer', 'name']),
  path(['card', 'holder_name'])
)
```

Reply:
boa, vou corrigir =)

Reply:
ajustado @marisamodolo